addons/website/static/src/js/menu/content.js

```diff
@@ -645,8 +644,8 @@ var EditMenuDialog = weWidgets.Dialog.extend({
                     'to_delete': this.toDelete,
                 }
             ],
-        }).then(function () {
-            return _super();
+        }).then(() => {
+            this.trigger("save");
```

Comment:
@younn-o This would need a huge comment to explain what is going on. However I'd prefer to handle that another way as not calling `_super` is a very bad practice (custo may `include` the default `Dialog` behaviors and would not expect a `_super` to not be called anymore (especially in stable). Instead, change the parent implementation of the `save` to do the `close` conditionally (e.g. a parameter "closeOnSave" for example) and set it to false for this specific `Dialog`. That way you'll have the same behavior for our codebase but it is still properly extendable.

Actually changing the behavior of not closing the dialog in stable is not entirely ok as custom code may rely on that too. For the sake of being entirely stable:
- Restore your custom class addition stuff for this fix in 15.0
- I'll disable forward-port of this fix
- Once it gets merged, cherry-pick it with X-original-commit mention inside the original PR's forward-port at https://github.com/odoo/odoo/pull/81087 and modifhy that commit to do what I explained here with the `closeOnSave` parameter.

Reply:
Ok :+1: